app/assets/javascripts/hyrax/app.js

```diff
@@ -40,6 +41,14 @@ Hyrax = {
       var controls = new CollectionControls($('#collection-controls'));
     },
 
+    // Collection types
+    collectionTypes: function() {
+      var element = $('.collection-types-wrapper')
+      if (element.length > 0) {
+        var CollectionTypes = require('hyrax/collection_types');
```

Comment:
There are requires all through this file that follow the same pattern.  Now that doesn't necessarily mean it is the best approach.  @jeremyf do you have an alternative suggestion?